portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,22 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Stephanie Ginting</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div style="text-align: center;" class="fade-in-image">
+            <img src="/images/profile-circle.png" width="200" />
+        </div>
+      <h1>STEPHANIE GINTING</h1>
+      <p>Computer Science student at Cornell University</p>
+      <!-- <p>Click <a href="/hello">here</a> to see content created by server code.</p> -->
```

Comment:
Typically when we submit code we try to avoid having commented sections like this. Seeing it in a repo is confusing as we don't know what the intent is.  I'd suggest either removing it or adding a comment explaining why this line is included but not used at the moment.

Reply:
Got it! I'll add new commit with a comment explaining why it's not used at this moment